contributors/design-proposals/scheduling/nodeaffinity.md

```diff
@@ -51,11 +51,14 @@ type NodeSelector struct {
 	NodeSelectorTerms []NodeSelectorTerm `json:"nodeSelectorTerms,omitempty"`
 }
 
-// An empty node selector term matches all objects. A null node selector term
-// matches no objects.
+// A null or empty node selector term matches no objects. The requirements of
+// them are ANDed.
```

Comment:
In our implementation, `MatchExpressions` and `MatchFields` are ORed. So, this comment is confusing. We should make it clear that individual terms of `MatchExpressions` are ANDed and so are the terms of `MatchFields`, but the result of the all terms of `MatchExpressions` are ORed with the results of all terms of `MatchFields`.

Reply:
The `MatchExpressions` and `MatchFields` within one `NodeSelectorTerm` are ANDed,  term are ORed. It also matches with our implementation :)

Reply:
Yes, that's right. Could you put this in the comment?